studio/next.config.mjs

```diff
@@ -81,4 +114,77 @@ const config = {
   },
 };
 
-export default withMarkdoc({ mode: "static" })(config);
+const withOptionalSentryConfig = (org, project, config) =>
+  withSentryConfig(config, {
+    // For all available options, see:
+    // https://github.com/getsentry/sentry-webpack-plugin#options
+
+    org: org,
+    project: project,
+    sourcemaps: {
+      disable: false,
+      deleteSourcemapsAfterUpload: true,
+    },
+
+    telemetry: false,
+
+    // Only print logs for uploading source maps in CI
+    silent: !process.env.CI,
+
+    // For all available options, see:
+    // https://docs.sentry.io/platforms/javascript/guides/nextjs/manual-setup/
+
+    // Upload a larger set of source maps for prettier stack traces (increases build time)
+    widenClientFileUpload: true,
```

Comment:
How big is the difference in build time? Do we need it at all?

Reply:
I made a test on a vercel test project. The build without the widen client file upload took 5 seconds longer. I would not say this is representative  but, it seems to not influence the build times significantly.

Reply:
ok, thanks for testing.